CHANGELOG.md

```diff
@@ -9,17 +9,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [`x.y.z`] - Unreleased
 
+### Breaking changes:
+- The logic for sending Spatial position updates has been changed, the `PositionUpdateFrequency` and the `PositionDistanceThreshold` have now been replaced in the GDK settings by 4 new variables. To update your project:
```

Comment:
Nit - I'd say "condition" instead of "logic" maybe. The important part o a user is that they configure things differently.